spec/atom-reporter.coffee

```diff
@@ -197,13 +197,27 @@ class AtomReporter
     time = "0#{time}" if time.length < 3
     @time.textContent = "#{time[0...-2]}.#{time[-2..]}s"
 
+  specTitle: (spec) ->
+    s = spec.suite
+    parentDescs = while s = s.parentSuite
```

Comment:
This loop's a bit confusing – mind refactoring this so all these assignments aren't inlined?